viewing_party/party.py

```diff
@@ -1,23 +1,180 @@
 # ------------- WAVE 1 --------------------
 
+from tests.test_constants import MOVIE_TITLE_1
+
+movie_dictionary = {}
+
 def create_movie(title, genre, rating):
-    pass
+    if title and genre and rating:
+        movie_dictionary["title"] = title
+        movie_dictionary["genre"] = genre
+        movie_dictionary["rating"] = rating
+        return movie_dictionary
+    else:
+        return None
+
+def add_to_watched(user_data, movie):
+    user_data = {"watched":[movie]}
+    return user_data
+
+def add_to_watchlist(user_data, movie):
+    user_data = {"watchlist": [movie]}
+    return user_data

```

Comment:
These two functions have a bug that our tests don't catch. If the `user_data` has content, this function will overwrite the dictionary.  The `user_data` dictionary could have key-value pairs for `watched`, `watchlist`, `friends`, etc.
```suggestion
def add_to_watched(user_data, movie):
    user_data["watched"].append(movie)
    return user_data

def add_to_watchlist(user_data, movie):
    user_data["watchlist"].append(movie)
    return user_data

```